### ISSUE TITLE
feature: rename Getting Started -> Examples to Getting Started -> Install KubeStellar

### ISSUE THREAD

**Issue by clubanderson** [kubestellar/kubestellar#2236]
### Feature Description

I am frustrated when I can't find a place on the KS site that simply says "Install KubeStellar" - because that's what I really want to do

### Proposed Solution

rename "Examples" to "Install KubeStellar" and update language on page to the same effect.

### Want to contribute?

- [ ] I would like to work on this issue.

**Comment by MikeSpreitzer** [comment]
I agree with that frustration. I do not think that we can fix it by simply renaming this document, because it has other content too. I recently wrote up my thoughts about this at https://github.com/kubestellar/kubestellar/pull/2229#discussion_r1649126413

**Comment by KPRoche** [comment]
See if what you are looking for is easier to find in the new structure (previewable at kproche.github.io/](https://kproche.github.io/kubestellar/doc-new-v2 )
(While this has been merged into main, it won't appear at kubestellar.io until the next release, as @MikeSpreitzer explained it to me)

**Comment by MikeSpreitzer** [comment]
The latest preview is in https://mikespreitzer.github.io/kcp-edge-mc/doc-update-621b/direct/usage-basics/ ; it was produced as part of #2239 .

**Comment by MikeSpreitzer** [comment]
I think #2240 would make our lives better.

**Comment by MikeSpreitzer** [comment]
Also, I realized that anyone can render `main` on their own fork. I just did that on mine: see https://mikespreitzer.github.io/kcp-edge-mc/main/readme/
